yogstation/code/datums/traits/neutral.dm

```diff
@@ -0,0 +1,53 @@
+/* Yogstation Neutral Traits */
+/datum/quirk/greyscale_vision
+	name = "Old School Vision"
+	desc = "Did you get stuck in an old video recorder? You can only see in black and white!"
+	value = 0
+	gain_text = "<span class='notice'>...Huh? You can't see colour anymore!</span>"
+	lose_text = "<span class='notice'>You can see colour again!</span>"
+
+/datum/quirk/greyscale_vision/add()
+	var/mob/living/carbon/human/H = quirk_holder
+	if(H)
+		H.add_client_colour(/datum/client_colour/greyscale)
+
+/datum/quirk/greyscale_vision/remove()
+	var/mob/living/carbon/human/H = quirk_holder
+	if(H)
+		H.remove_client_colour(/datum/client_colour/greyscale)
+
+/datum/quirk/inverted_vision
+	name = "Inverted Colour Vision"
+	desc = "You see red and blue colours as reversed."
+	value = 0
+	gain_text = "<span class='notice'>You feel like you're seeing colours differently.</span>"
+	lose_text = "<span class='notice'>You feel like you're seeing colours normally again.</span>"
+
+/datum/quirk/inverted_vision/add()
+	var/mob/living/carbon/human/H = quirk_holder
+	if(H)
+		H.add_client_colour(/datum/client_colour/inverted)
+
+/datum/quirk/inverted_vision/remove()
+	var/mob/living/carbon/human/H = quirk_holder
+	if(H)
+		H.remove_client_colour(/datum/client_colour/inverted)
+
+/datum/quirk/waddle
+	name = "Waddle"
+	desc = "You have an extra bounce in your step."
+	value = 1
```

Comment:
honestly, having the waddle component means it's harder to change identity, whether it's via changeling transform or via chameleon kit, so maybe it should be 0 points or negative. *Or maybe a genetics mutation*